sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/npm-log-level/build.sbt

```diff
@@ -0,0 +1,21 @@
+name := "npm-log-level"
+enablePlugins(ScalaJSBundlerPlugin)
+scalaVersion := "2.11.12"
+webpackBundlingMode := BundlingMode.LibraryOnly()
+emitSourceMaps := false
+npmDependencies in Compile += "http-aws-es" -> "6.0.0"
+
+val assertWarning = taskKey[Unit]("checks npm log  warn")
+assertWarning := check(true).value
+
+def check(expectation: Boolean) = Def.task[Unit] {
```

Comment:
Is this `expectation` parameter needed?

Reply:
Actually, this is copy from sbt's test https://github.com/sbt/sbt/blob/cc0dc0c9037eea25ff77d1081a9c4584823fcab3/sbt/src/sbt-test/reporter/nowarn/build.sbt#L9-L20

Since it is enough to check that log should contain `[warn]`, I will remove the unnecessary parameter.